packages/rfw/example/wasm/pubspec.yaml

```diff
@@ -1,10 +1,11 @@
 name: rfw_wasm
 description: Example of using Wasm with RFW
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
-version: 1.0.0+1
+version: 1.0.0+2
 
 environment:
   sdk: ">=2.13.0 <3.0.0"
+  flutter: ">=3.0.0"
```

Comment:
The Flutter constraints need to go in the top-level package rather than the examples, as that's what the legacy analysis is looking at. Analysis runs over the whole package at once, not individual subpackages.

Reply:
Gotcha, done